$OpenBSD$

cparser doesn't understand __attribute__((__gnu_inline__)).
This causes all sorts of multiple definition linker errors.
So let's set the GCC compatibility to too old for it.

Index: src/driver/predefs.c
--- src/driver/predefs.c.orig
+++ src/driver/predefs.c
@@ -246,7 +246,7 @@ void add_predefined_macros(void)
 
 	/* let's pretend we are a GCC compiler */
 	add_define("__GNUC__",            "4", false);
-	add_define("__GNUC_MINOR__",      "6", false);
+	add_define("__GNUC_MINOR__",      "1", false);
 	add_define("__GNUC_PATCHLEVEL__", "0", false);
 	if (dialect.cpp)
 		add_define("__GNUG__", "4", false);
